tests/e2e/standard/test_init.py

```diff
@@ -122,6 +130,24 @@ def test_upload_dependency_file(self, environment):
             with zipped.open(filename, "r") as file:
                 assert file.read().decode(encoding="utf-8") == "some-dependency==1.0.0"
 
+    def test_tracking_uncommitted_changes(self, environment):
+        try:
+            # local setup
+            repo = Repo(".")
+        except InvalidGitRepositoryError:
+            # CI
+            repo = Repo.init()
+
+        with open("some_file.txt", "w") as fp:
+            fp.write("some-content\n")
+
+        repo.git.add("some_file.txt")
+
+        assert repo.is_dirty()
+        with neptune.init_run(project=environment.project) as run:
+            run.sync()
+            assert run.exists("source_code/diff")
```

Comment:
How about checking the content of a diff? We prepared a `some_file.txt` so it shouldn't be that hard to achieve :wink: